index.html

```diff
@@ -81,23 +117,54 @@ <h1>Multi-Post<br>Stories</h1>
                     <li><p>2015</p></li>
                 </ul>
             </div>
-            <p class="contents">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <p class="content">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
             <br>
             <div class="buttons">
                 <ul class="list">
                     <li><div class="tag">html</div></li>
                     <li><div class="tag">css</div></li>
                     <li><div class="tag">javascript</div></li>
                 </ul>
+                <button class="active">See Project</button>
```

Comment:
- [ ] Kindly change the color for the `see project` button to get similar colors with see project buttons
![port44](https://user-images.githubusercontent.com/55635977/130281944-a4a65105-fe19-49eb-bb59-2b4a645cdf6d.png)

Reply:
Hello Emmy.  Thank you for the guidance. It is highly appreciated. However, the color of the third card is actually like that on the Figma project